schematypes.go

```diff
@@ -28,6 +28,27 @@ type Message interface {
 	GetSeq() int
```

Comment:
Makes me wonder if similarly this should have GetProtocolMessage()

Reply:
Just to make sure this is not overlooked: changing Message would be a breaking change for any external type that implements the current definition of the interface.

Reply:
GetProtocollMessage() would make more sense then GetSeq() on the message-interface.
But changing the existing message-interface is sadly not an option because of the compatibility-issues that would arise. 
One idea would be to introduce a secondary message-interface (GenericMessage?) that has the GetProtocollMessage() method, but I fear that having two seperate Message-Interfaces would just lead to confusion.